avm/client/src/avmHelpers.ts

```diff
@@ -27,6 +27,7 @@ const tetrapletRepr = new MsgPackRepr();
 const callResultsRepr = new MulticodecRepr(new MsgPackRepr());
 //
 const defaultAquaVMRuntimeMemoryLimit = Number("4294967296");
+const aquaVMRuntimeHardLimitsAreDisabled = Boolean(false);
```

Comment:
Why not just put values directly?
```suggestion
const defaultAquaVMRuntimeMemoryLimit = 4294967296;
const aquaVMRuntimeHardLimitsAreDisabled = false;
```

Reply:
Changed